build-system/app-index/template.html

```diff
@@ -208,13 +212,50 @@ <h1 class="amp-logo">AMP</h1>
   </wrap>
   <div class="file-list-container">
     <wrap>
-      <h3 class="code"><!-- basepath --></h3>
+      <h3 class="code" id="basepath"><!-- basepath --></h3>
       <div class="push-right-after-heading">
+        <form hidden id="examples-mode-form">
+          <label for="examples-mode-select">
+            Document mode:
+            <select id="examples-mode-select">
+              <option value="/" selected>standard</option>
+              <option value="/a4a/">a4a</option>
+              <option value="/a4a-3p/">a4a-3p</option>
+              <option value="/inabox/1/">inabox</option>
+            </select>
+          </label>
+        </form>
         <a href="/~" class="underlined">List root directory</a>
       </div>
       <ul class="file-list"><!-- listing --></ul>
     </wrap>
   </div>
+  <script>
+    // TODO(alanorozco): Write this in ES2015 to be transpiled.
+    function maybeEnableExampleModeSelect() {
```

Comment:
Would say to just write this as an IIFE, but since we are going to move this anyways soon should be fine 😄